pipeline/pipeline/tasks/dropbox_utils.py

```diff
@@ -0,0 +1,111 @@
+from io import StringIO
+import re
+import datetime
+from luigi.contrib.dropbox import DropboxTarget
+from pipeline.config import DROPBOX_TOKEN
+import dropbox
+
+
+def dropbox_target(path):
+    return DropboxTarget(path, DROPBOX_TOKEN)
+
+
+def textio2stringio(text_io_wrapper):
+    """Converts the contents of a text io wrapper to a StringIO instance
+
+    Pandas doesn't accept a TextIOWrapper because it expect that it will have the name kwarg set,
+    we can't do that since we're using dropbox to store files and the client download content as bytes
+    (https://luigi.readthedocs.io/en/stable/_modules/luigi/contrib/dropbox.html#DropboxTarget.temporary_path),
+    however pandas accepts an StringIO instances.
+    """
+    return StringIO(text_io_wrapper.read())
+
+
+def textio2binary(text_io_wrapper):
+    return text_io_wrapper.buffer.read()
+
+
+# ? ==============================
+# ? dropbox_clear_space
+# ? Additions re: Dropbox clear space
+def ensure_available_space():
```

Comment:
this method could do with some smaller helper functions (that could have been unittested).
But let's instead invest our time making dropbox obsolete :-)
Nice work.

Reply:
sounds good, thanks!